### DISCUSSION TITLE
Crash with 0.108.0 when NC_PUBLIC_URL is set [May 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5844] [Discord Threads]
> 🙋‍♂️ **sasch90**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Fixed the bug, will rollup a bug fix release in next couple hours - https://github.com/nocodb/nocodb/issues/5778  
#  
> 👨‍💻 **pranavxc**  
#### https://github.com/nocodb/nocodb/releases/tag/0.108.1  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1113126831164498031)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
